frontend/e2e-tests/page-objects/crud/PollingStationListPgObj.ts

```diff
@@ -0,0 +1,15 @@
+import { type Locator, type Page } from "@playwright/test";
+
+export class PollingStationListPgObj {
+  protected readonly page: Page;
+
+  readonly alert: Locator;
+  readonly createPollingStationButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.alert = this.page.locator("role=alert");
+    this.createPollingStationButton = this.page.getByRole("button", { name: "Stembureau toevoegen" });
```

Comment:
```suggestion
    this.alert = page.locator("role=alert");
    this.createPollingStationButton = page.getByRole("button", { name: "Stembureau toevoegen" });
```

To make it consistent with the other page objects and the Playwright documentation. I have no idea if it makes a difference in any other way.

Reply:
it makes a difference, it's easier to copy paste to another method/function with `this` but I've removed it since consistency is more important :)